### ISSUE TITLE
Android: file shared from Memories is not recognized as image by system share dialog

### ISSUE THREAD

**Issue by allista** [pulsejet/memories#1041]
**Describe the bug**

When I try to process a photo from Memories with another native app on Android I cannot directly share the photo to any image-handling app, only to those that handle all file types (file managers, messengers, etc.)

On the other hand, when I _save_ the photo to a file and "open with"  it from a file manager, I see all the apps capable of image handling.

**To Reproduce**

- on Android phone/tablet have at least one image handling app (e.g. Snapseed)
- open Memories
- open any photo (jpeg, png, raw)
- press the "share" button
- select "original file" option
    - "low res" and "high res" options has the same effect
- observe the list of apps that can handle the share request
- observe that the image handling apps are not in the list, only file managers and messengers

**Screenshots**

![image](https://github.com/pulsejet/memories/assets/6084589/00429d9a-0885-42df-8acc-7c94c472bef2)

![image](https://github.com/pulsejet/memories/assets/6084589/bea5008d-0b0f-4bd5-be13-dc6919daa122)

**Platform:**

- Android app
- OS: Linux 5.15.0-88-generic x86_64
- Memories Version: 6.2.2
- Nextcloud Version: 27.1.5
- PHP Version: 8.1.27

**Comment by Forza-tng** [comment]
See also https://github.com/pulsejet/memories/issues/1038#issue-2127520509

**Comment by allista** [comment]
I was not sure if it is about the same thing, or the other way around, that is having Memories as a target to share files to.
Feel free to merge them if they are.

**Comment by Forza-tng** [comment]
> I was not sure if it is about the same thing, or the other way around, that is having Memories as a target to share files to. Feel free to merge them if they are.

No, that ticket is about using Memories as file picker from other apps. For example if I were to attach a file to this ticket, I can choose Nextcloud, Files, Gallery, etc as file picker, but not Memories.

**Comment by pulsejet** [comment]
https://github.com/pulsejet/memories/commit/8597d27999a1ccb8273083f417afc52935bd50ec
